### PR TITLE
button: Move keyboard related stuff into common.blocks (close #359)

### LINKED ISSUE #359: button: Move keyboard related stuff into common.blocks

### PR DESCRIPTION
**Pull request by dfilatov**
cc @narqo @veged

### CODE REVIEW

File: common.blocks/button/button.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 modules.define(
     'button',
-    ['i-bem__dom', 'jquery', 'dom'],
-    function(provide, BEMDOM, $, dom) {
+    ['i-bem__dom', 'jquery', 'dom', 'functions'],
+    function(provide, BEMDOM, $, dom, functions) {
+
+var KEY_CODE_SPACE = 32,
+    KEY_CODE_ENTER = 13;
```
Comment (narqo) [inline common.blocks/button/button.js:7]: as we don't not fix our bem-core's version in bem-components, maybe we could use `keyboard__codes` from current v2's `HEAD`?

File: common.blocks/button/button.spec.js [hunk 2]
```diff
@@ -61,21 +61,37 @@ describe('button', function() {
...
+            button.hasMod('pressed').should.be.true;
+        });
+
+        it('should not be pressed on pointerrelease', function() {
             button.domElem.trigger('pointerrelease');
             button.hasMod('pressed').should.be.false;
         });
 
-        it('should not set "hovered" mod if it has "disabled" mod', function() {
+        it('should not be pressed if it is disabled', function() {
```
Comment (narqo) [inline common.blocks/button/button.spec.js]: Guess you could say "(it) should not be pressed if disabled" :)

**Comment by narqo** [comment]
looks good

File: common.blocks/button/button.js (hunk 1 above)
Comment (dfilatov) [inline common.blocks/button/button.js:7]: I don't understand how bower resolve that case. I've tried:

```
rm -rf libs
bower-npm-install
```

I've got:

```
bower bem-core#v2             validate 2.1.0 against git://github.com/bem/bem-core.git#v2
```

WTF? Why has bower installed 2.1.0?

File: common.blocks/button/button.spec.js (hunk 2 above)
Comment (dfilatov) [inline common.blocks/button/button.spec.js]: fixed

File: common.blocks/button/button.js (hunk 1 above)
Comment (narqo) [inline common.blocks/button/button.js:7]: Magic :(

**Pull request merged**